src/System.Management.Automation/engine/CommandParameter.cs

```diff
@@ -15,12 +15,14 @@ internal sealed class CommandParameterInternal
     {
         private class Parameter
         {
+            internal Ast ast;
             internal IScriptExtent extent;
```

Comment:
Adding a new parameter will bloat the **generated** code significantly - this is an extra parameter for every argument on every command invocation.  We should avoid that if there are alternatives.